Source/Models/FORMTarget.h

```diff
@@ -18,20 +19,22 @@ typedef NS_ENUM(NSInteger, FORMTargetActionType) {
     FORMTargetActionNone
 };
 
-@interface FORMTarget : NSObject
+@interface FORMTarget : FORMFieldElement
 
 @property (nonatomic, copy) NSString *targetID;
-@property (nonatomic, copy) id targetValue;
+@property (nonatomic, copy) id value;
 @property (nonatomic, copy) NSString *typeString;
 @property (nonatomic, copy) NSString *actionTypeString;
 @property (nonatomic, copy) NSString *condition;
 
-@property (nonatomic) FORMFieldValue *value;
+@property (nonatomic) FORMFieldValue *fieldValue;
```

Comment:
Can we add backwards compatibility? Might be nice if we don't have to jump to Form 4.0.0

You can just make `fieldValue` point to `value` with the deprecated flag.

```objc
- (NSInteger)foo:(NSInteger)bar __attribute__((deprecated("Use fooWithBar: instead")));
```